app/views/events/participants.html.haml

```diff
@@ -0,0 +1,24 @@
+.col-md-12
+  %h3 Participants of #{@event.name}
+  %br
+  %table.table
+    %tr
+      %th #
+      %th Nickname
+      %th Email
+      %th State
+      %th Role
+      %th Change role to event organizer
+    - @event.requests.map(&:user).uniq.each_with_index do |user,index|
```

Comment:
@ChrisBr thanks you :), I didn't knew about the eager loading associations.

Before it was taking 12 queries now it is taking only 1 after changing it to :
`User.includes(:requests, :profile).where(requests: {event: @event})`